src/test/effectslottest.cpp

```diff
@@ -1,80 +0,0 @@
-#include <gtest/gtest.h>
-#include <gmock/gmock.h>
-
-#include <QtDebug>
-#include <QScopedPointer>
-
-#include "mixxxtest.h"
-#include "control/controlobject.h"
-#include "effects/effectchain.h"
-#include "effects/effectchainslot.h"
-#include "effects/effectsmanager.h"
-#include "effects/effectmanifest.h"
-#include "effects/effectrack.h"
-#include "effects/effectslot.h"
-
-#include "test/baseeffecttest.h"
-
-using ::testing::Return;
-using ::testing::_;
-
-class EffectSlotTest : public BaseEffectTest {
```

Comment:
This is unrelated to the work you've done so far, so leave these tests in for now. When we consolidate Effect & EffectSlot, then let's remove these tests.